components/measurement/nettests/WebConnectivity.js

```diff
@@ -167,32 +167,48 @@ FailureString.propTypes = {
   failure: PropTypes.string
 }
 
+const DnsNarrowAnswerCell = (props) => (
+  <Box width={1/12}>{props.children}</Box>
+)
+
 const DnsAnswerCell = (props) => (
-  <Box width={1/8}>{props.children}</Box>
+  <Box width={1/4}>{props.children}</Box>
 )
 
 DnsAnswerCell.propTypes = {
   children: PropTypes.any
 }
 
-const FiveColRow = ({ name = 'Name', netClass = 'Class', ttl = 'TTL', type = 'Type', data = 'DATA', header = false}) => (
+const dnsAnswerIpInfo = (dnsAnswer) => {
+    return dnsAnswer.asn
+        ? dnsAnswer.as_org_name
+        ? `AS${dnsAnswer.asn} (${dnsAnswer.as_org_name})`
+        : `AS${dnsAnswer.asn}`
+    : dnsAnswer.as_org_name
+        ? `Unknown AS (${dnsAnswer.as_org_name})`
```

Comment:
We are in the process of fully localizing Explorer, so the hardcoded strings should be moved to the localization file (https://github.com/ooni/explorer/blob/master/public/static/lang/en.json) and then added to the code with either `intl.formatMessage` or `<FormattedMessage />`